getting-started/versioning.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Versioning"
+weight: 100
+---
+
+# Versioning
+
+### Format
+
+`v<major>.<minor>`
+
+examples: `v0.1` `v0.2`
+
+All components of this API have single, common version.
```

Comment:
All components of this API have single, common version. -> All components of this API have a single common version.

Reply:
pojawiła się wątpliwość, ze jak to zdanie znajdzie się w sekcji Format to może zostać źle zrozumiane, stąd przenoszę to do wstępu